src/foam/nanos/fs/fileDropZone/FileCard.js

```diff
@@ -97,12 +98,20 @@ foam.CLASS({
     },
     {
       name: 'index'
+    },
+    {
+      name: 'selected'
     }
   ],
 
   methods: [
     function initE() {
       this.SUPER();
+      if ( this.selected == this.index ) {
+        this.style({
+          "border-color": "blue"
```

Comment:
Is it possible to use: 
/* PRIMARY1 */ #hexcode  (or whatever is appropriate)
as we do elsewhere? so the colour is configurable and matches.

Reply:
Updated. Is /* PRIMARY1 */ just a comment, or it is meaningful for foam parser?

Reply:
/*%PRIMARY1%*/ works. If theme CSS variable PRIMARY1 is available, it goes with PRIMARY1. If it is not available, /*%PRIMARY1%*/ will work like a comment.